lua/codeium/config.lua

```diff
@@ -5,6 +5,7 @@ function M.defaults()
 		manager_path = nil,
 		bin_path = vim.fn.stdpath("cache") .. "/codeium/bin",
 		config_path = vim.fn.stdpath("cache") .. "/codeium/config.json",
+		override_download_url = "https://github.com",
```

Comment:
Can we call this `mirror_domain`?

Reply:
honestly mirror_domain sounds more complicated. I actually like the override_download_url better because it tells you, kind of, that a change is happening. its when you want to change where.. something.. is downloaded. it requires less inference about it's purpose, but, in fact I'd even like it more clear, something like language_server_download_url. then people know exactly what download is being referred to, because otherwise, it could be initially confusing. what download? my completions? the language server? what is it?

Reply:
Works for me!

Reply:
Thanks! It is indeed more reasonable to change it to `language_server_download_url`.